README.md

```diff
@@ -467,6 +467,14 @@ Manages the GPG keys that Apt uses to authenticate packages.
 
 This module is tested and officially supported on Debian 6 and 7 and Ubuntu 10.04, 12.04, and 14.04. Testing on other platforms has been light and cannot be guaranteed.
 
+This module is not designed to be split across stages. If you want to ensure that apt update is run before all packages, add a proper dependency like this:
+
+~~~puppet
+Class['apt::update'] -> Package<||>
+~~~
+
+Be aware that this conflicts with virtual package resources.
```

Comment:
I understand.

What about:
*Be aware that this example will realize all your virtual package resources. Adapt it to your environment.*?